### PR TITLE
Create ttmeiju.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

File: src/chrome/content/rules/ttmeiju.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	<target host="www.ttmeiju.com"/>
+
+	<!--	Mixedcontent on home pages	-->
+	
+	<exclusion pattern="^http://(www\.)?ttmeiju\.com/$" />
+	<exclusion pattern="^http://bbs\.ttmeiju\.com/$" />
+	<exclusion pattern="^http://bbs\.ttmeiju\.com/forum\.php/$" />
+		<test url="http://bbs.ttmeiju\.com/forum.php/" />
+	<exclusion pattern="^http://bbs\.ttmeiju\.com/forum\.php.+" />
+		<test url="http://bbs.ttmeiju.com/forum.php?mod=forumdisplay&fid=38" />
```
Comment (fuglede) [inline src/chrome/content/rules/ttmeiju.xml]: The ampersands need to be replaced by <code>&amp;</code>.

**Comment by renyouguo** [comment]
@fuglede thanks，done.

**Comment by J0WI** [comment]
Please also add test urls without the www prefix if you allow both in the regex.

File: src/chrome/content/rules/ttmeiju.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	<target host="ucenter.ttmeiju.com" />
+	<target host="www.ttmeiju.com" />
+
+	<!--	Mixedcontent on home pages	-->
+	
+	<exclusion pattern="^http://(www\.)?ttmeiju\.com/$" />
+	
+	<exclusion pattern="^http://(www\.)?ttmeiju\.com/search\.php" />
+		<test url="https://ttmeiju.com/search.php?keyword=y&amp;range=0" />
+		<test url="https://www.ttmeiju.com/search.php?keyword=y&amp;range=0" />
```
Comment (J0WI) [inline src/chrome/content/rules/ttmeiju.xml]: test urls should be http: (see Travis)

**Pull request merged**